kart/tabular/export.py

```diff
@@ -0,0 +1,304 @@
+import os
+import re
+
+import click
+from osgeo import gdal, ogr
+
+from kart.cli_util import (
+    call_and_exit_flag,
+    KartCommand,
+)
+from kart.completion_shared import repo_path_completer
+from kart.crs_util import make_crs
+from kart.exceptions import CrsError
+
+
+def is_writable_vector_format(driver):
+    m = driver.GetMetadata()
+    return m.get("DCAP_VECTOR") == "YES" and m.get("DCAP_CREATE") == "YES"
+
+
+def list_export_formats(ctx):
+    """List the supported export formats as reported by gdal."""
+
+    for i in range(ogr.GetDriverCount()):
```

Comment:
since GDAL 2.0 [vector and raster drivers are unified](https://gdal.org/development/rfc/rfc46_gdal_ogr_unification.html) (to support mixed formats); we should be using `gdal.*` objects/methods for drivers & datasets if we're writing new code.